recipe-app/src/App.js

```diff
@@ -1,25 +1,29 @@
 import logo from './logo.svg';
 import './App.css';
+import RecipeList from "./RecipeList";
+import RecipeForm from "./RecipeForm";
+import {Redirect, Route, Switch} from "react-router-dom";
+import NavBar from "./NavBar";
+import RecipeInfo from "./RecipeInfo";
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    return (
```

Comment:
Consider adding prettier to your IDE so to automatically format your code (you'll use it a lot in our repos)